addons/lunch/lunch.py

```diff
@@ -311,6 +190,7 @@ def fields_view_get(self, cr, uid, view_id=None, view_type=False, context=None,
                                     ('partially','Partially Confirmed')] \
                                 ,'Status', readonly=True, select=True, copy=False),
         'alerts': fields.function(_alerts_get, string="Alerts", type='text'),
+        'previous_order_details': fields.function(_lunch_order_get, string="Previous Orders", type='char'),
```

Comment:
It is not, but it is not recommended to use them because it avoids you to make a good model design :)